src/client/components/HomePage/LandingGraphicSliver.jsx

```diff
@@ -102,10 +102,10 @@ const useStyles = makeStyles((theme) =>
 
 const LandingGraphicSliver = () => {
   const classes = useStyles()
-  const topPaddingMultipler = () => {
-    const theme = useTheme()
-    const isMediumWidth = useMediaQuery(theme.breakpoints.up('md'))
-    const isDesktopWidth = useMediaQuery(theme.breakpoints.up('lg'))
+  const useTopPaddingMultipler = () => {
+    const theme = useTheme() // eslint-disable-line react-hooks/rules-of-hooks
+    const isMediumWidth = useMediaQuery(theme.breakpoints.up('md')) // eslint-disable-line react-hooks/rules-of-hooks
+    const isDesktopWidth = useMediaQuery(theme.breakpoints.up('lg')) // eslint-disable-line react-hooks/rules-of-hooks
```

Comment:
Given that the hooks are being used within a custom hook, `eslint-disable-line react-hooks/rules-of-hooks` should be unnecessary
```suggestion
    const theme = useTheme()
    const isMediumWidth = useMediaQuery(theme.breakpoints.up('md'))
    const isDesktopWidth = useMediaQuery(theme.breakpoints.up('lg'))
```

Reply:
Oh right right @LoneRifle